prime-router/docker-compose.yml

```diff
@@ -68,6 +68,32 @@ services:
     networks:
       - prime-router_build
 
+
+  # local FTPS Server
+  ftps:
+    image: stilliard/pure-ftpd
+    ports:
+      - "21000:21"
+      - "30000-30009:30000-30009"
+    volumes:
```

Comment:
I think it would make sense to make the FTP storage area a bind mount; that will make it easy for folks to actually look at what's inside the FTP dump location from outside of the container without having to FTP or bash into it themselves (i.e. they can just look on their own filesystem).

Reply:
Would it be acceptable to create more empty directories within the project to back this bind mount? 

For testing I just used lftp to observe contents, and that worked at least when run locally.

Reply:
I'd put them in a `./prime-router/build/ftps` directory; if you look at the `docker-compose.yml`:L53 file, we do a similar thing for azurite where we bind-mount `.prime-router/build/azurite`.
If it's part of the `build` directory, it'll be automatically git-ignored